### ISSUE TITLE
Implement better 2d phone controls

### ISSUE THREAD

**Issue by gfodor** [mozilla/mr-social-client#84]
the current "dual axis" controls on mobile work but are not ideal. we should implement the control scheme where single finger drag orbits camera and dual finger pinch-to-zoom moves forwards and backwards.

**Comment by robertlong** [comment]
The pinch to zoom control scheme seems like it would require some sort of tutorial. It's not immediately obvious how to move forward and backward with that control scheme.

**Comment by gfodor** [comment]
yeah I agree a simple popup overlay until the user activates the feature should suffice.

**Comment by gfodor** [comment]
Specifically here is the request for this ticket:

- Touching and dragging on the screen above the stick area rotates the camera

- The gyro, if available, rotates the camera but at small ratio of real world to virtual world radians. Gross head movement should rely upon user touch.

- Pinch and un-pinch to move forward/backward

- Ensure interactables work

- Sticks should still work if you thumb down on the bottom of the screen as usual

- Stretch goal: simple tooltip before user first uses the pinch action to instruct them about it

Use cases to ensure work properly:

- I'm using my phone flat on my desk, but can still navigate the space easily
- I'm using my phone held at an angle in my hand, but can still navigate easily
- I'm using my phone held straight up, and want to be able to "look at" people

**Comment by johnshaughnessy** [comment]
I was looking considering taking on hammer.js as a dependency, but the basic examples are not working in chrome. https://github.com/hammerjs/hammer.js/issues/1130
